### DISCUSSION TITLE
How to give dynamic padding to mark node?

### DISCUSSION THREAD

**Discussion by Proful** [code-hike/v1#20] [Q&A]
The container padding is dynamic.

How to match mark's padding?
![image](https://github.com/user-attachments/assets/0f4ca0f9-0334-4ac8-98a5-9579e593e192)

**Comment by pomber** [comment]
what I usually do is avoid adding horizontal padding to the container and add the padding in the `Line` component

**Reply by Proful** [reply]
In my use case, I have to dynamically change padding. I am aware of the line horizontal padding, but bit restrictive to change it at runtime. One option to pass as annotation query but that is not very intuitive. Any other way you suggest?

**Reply by pomber** [reply]
what if you change it using a CSS variable?
